docs/public-networks/reference/engine-api/index.md

```diff
@@ -224,6 +224,64 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"engine_forkchoiceUpdatedV1","par
 
 </Tabs>
 
+### `engine_getBlobsV1`
+
+Returns the blobs corresponding to the specified blob versioned hashes.
+
+#### Parameters
+
+`blobHashes`: *array* of *strings* - Array of blob versioned hashes
+
+#### Returns
+
+`engineGetBlobsResultV1`: *array* of *objects* - Array of blob and proof objects
+
+<Tabs>
+
+<TabItem value="curl HTTP" label="curl HTTP" default>
+
+```bash
+curl -X POST --data '{"jsonrpc":"2.0","method":"engine_getBlobsV1","params":[["0x000657f37554c781402a22917dee2f75def7ab966d7b770905398eba3c444014"]],"id":1}' http://127.0.0.1:8550
```

Comment:
Maybe add a second versioned hash, so it's obvious that this is a list?
Have you tested the curl command? Probably returns null, but should not complain.